compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/ImmutableTypeCloner.java

```diff
@@ -542,7 +542,7 @@ private static void populateImmutableStructureFields(Types types, SymbolTable sy
 
     private static String getPackageAlias(SymbolEnv env, String compUnitName, PackageID typePkgId) {
         for (BLangImportPackage importStmt : env.enclPkg.imports) {
-            if (!typePkgId.equals(importStmt.symbol.pkgID)) {
+            if (importStmt.symbol == null || !typePkgId.equals(importStmt.symbol.pkgID)) {
```

Comment:
Why do we need this change?

Reply:
when compiling `language-server/modules/langserver-core/src/test/resources/extensions/document/configurables/configurables.bal` gives symbol as null for `ballerina/io` import

Reply:
The original issue #30561

Reply:
Please add a test for that scenario too. We can add it separately.